src/Microsoft.Health.Fhir.Core/Features/Validation/AttributeValidator.cs

```diff
@@ -41,7 +42,10 @@ public IEnumerable<ValidationFailure> Validate(PropertyValidatorContext context)
                 {
                     foreach (var error in results)
                     {
-                        yield return new ValidationFailure(error.MemberNames?.FirstOrDefault(), error.ErrorMessage);
+                        ITypedElement typedElement = resourceElement.Instance;
+                        var fullFhirPath = typedElement.InstanceType + "." + error.MemberNames?.FirstOrDefault();
```

Comment:
The code `error.MemberNames?.FirstOrDefault()` reads that it could be a null value. In this case perhaps we just use the InstanceType?

Reply:
Makes sense, fixed in 2b34f6d.